### ISSUE TITLE
[Playbook] 3D Gaussian Splatting (3DGS) Scene Reconstruction with Pytorch+ROCm

### ISSUE THREAD

**Issue by charyang-ai** [amd/playbooks#21]
• Use Cases: 3D/4D Reconstruction, VR/AR/XR, Human avatar, Physical AI, World model (Feifei Li, Marble)  

• Hardware: AI MAX 395 with Radeon 8060S iGPU offers strong compute and unified memory for 3DGS.

• Training: Use mixed precision, batching, and gradient accumulation to fit large Gaussian sets efficiently.

• Memory: Unified memory avoids CPU–GPU transfers, improving stability and throughput.

• Rendering: Tile-based rasterization enables real-time Gaussian splat rendering.

Use cases: Suitable for end-to-end 3DGS training, rendering, and rapid prototyping on a single device.

<img width="806" height="454" alt="Image" src="https://github.com/user-attachments/assets/855fc029-16cb-4113-8ac6-68a8a4efade6" />

**Comment by bconsolvo** [comment]
Not prioritizing for now as it may not work on Halo, but will re-open if we decide to move forward with it.
